geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -307,6 +343,8 @@ pub(crate) fn load_plugin_from_config(
         libpath = config_dir.join(libpath);
     }
 
+    let libname = result["libname"].as_str().map(|s| s.to_owned());
```

Comment:
I think we should make it optional now. If it is set in the config file, use it, otherwise old behavior to keep it backward compatible. I would like to still call it "name" instead of libname, we do not care about the libname. The lib path has the libname anyway.

Reply:
Changed `libname` to `name`. Value is already optional.